### PR TITLE
Modify gce snapshot code

### PR DESCRIPTION
**Pull request by jingxu97**
modify gce snapshot code to support

### CODE REVIEW

File: snapshot/pkg/cloudprovider/providers/gce/gce.go [hunk 1]
```diff
@@ -2922,8 +2922,8 @@ func (gce *Cloud) getSnapshotByName(snapshotName string) (*gceSnapshot, error) {
 }
 
 // CreateSnapshot creates a snapshot
-// TODO: CreateSnapshot should return snapshot status
-func (gce *Cloud) CreateSnapshot(diskName string, zone string, snapshotName string, tags map[string]string) error {
+// it returns snapshot id and status
```
Comment (rootfs) [inline snapshot/pkg/cloudprovider/providers/gce/gce.go]: golint will complain

**Comment by rootfs** [comment]
@jingxu97 do you need anything in find snapshot and describe snapshot too?

**Comment by jingxu97** [comment]
@rootfs yes, working on findsnapshot

**Comment by tsmetana** [comment]
I can see some overlap with the newer PR #400. Should those be merged?

**Comment by jingxu97** [comment]
@tsmetana I think I will put all the changes in PR #400. So don't need to merge this one. Thanks!

**Comment by tsmetana** [comment]
Closing as per @jingxu97's last comment: the #400 has been merged.

**Pull request closed without merging**